fitlins/interfaces/afni.py

```diff
@@ -265,14 +265,24 @@ def save_remlfit_results(self, maps, contrasts, runtime):
         vol_labels = parse_afni_ext(maps["effect_size"])["BRICK_LABS"].split("~")
 
         effect_bool = np.array([x.endswith("Coef") for x in vol_labels])
+        clean_vol_labels = []
+        for ii, x in enumerate(vol_labels):
+            if (ii in [0, 1]):
+                clean_vol_labels.append(x)
+            elif x.endswith('_Coef') or x.endswith('_Tstat'):
+                clean_vol_labels.append('#'.join(x.split('#')[:-1]))
+            elif x.endswith('_R^2') or x.endswith('_Fstat'):
+                clean_vol_labels.append('_'.join(x.split('_')[:-1]))
```

Comment:
```suggestion
                clean_vol_labels.append(x.rsplit('_')[0])
```

Reply:
I think this should probably be x.rsplit('_', 1)[0] as well, right?